apps/frontend/src/components/form/GrantForm.tsx

```diff
@@ -0,0 +1,283 @@
+/* eslint-disable react/jsx-props-no-spreading */
+
+// do we add notes section to both grant form and medical form or just one
+import React from 'react';
+import Form, { FormValues } from './Form';
+import {
+  FormControl,
+  FormLabel,
+  Input,
+  NumberInputField,
+  NumberInput,
+  InputGroup,
+  InputLeftElement,
+  FormHelperText,
+  Textarea,
+} from '@chakra-ui/react';
+import { PhoneIcon } from '@chakra-ui/icons';
+import { Field, FieldProps } from 'formik';
+
+interface GrantFormValues {
+  childName: string;
+  dob: Date;
+  gender: string;
+  parentName: string;
+  addressStreet: string;
+  addressCity: string;
+  addressState: string;
+  addressZipcode: number;
+  phoneNumber: string;
+  cellPhoneNumber: string;
+  email: string;
+  grantDollarAmount: number;
+  useOfGrant: string;
+  parentSignature: string;
+  parentSignatureDate: Date;
+  grantFormNotes: string;
+}
+
+interface ExampleFormProps {
+  onSubmit?: (values: Partial<GrantFormValues>) => Promise<void>;
+}
+
+const alertOnSubmit = async (values: FormValues) => {
+  await new Promise((resolve) => setTimeout(resolve, 1000));
+  alert(JSON.stringify(values, null, 2));
+};
+
+const ExampleForm: React.FC<ExampleFormProps> = ({ onSubmit }) => (
+  <Form onSubmit={onSubmit ?? alertOnSubmit} initialValues={{}}>
+    <Field name="childName">
+      {({ field, form }: FieldProps) => (
+        <FormControl
+          isRequired
+          isInvalid={Boolean(
+            form.errors['childName'] && form.touched['childName']
+          )}
+        >
+          <FormLabel htmlFor={'childName'}>Child Name</FormLabel>
+          <Input {...field} id={'childName'} />
+        </FormControl>
+      )}
+    </Field>
+    <Field name="dob">
```

Comment:
sorry i know i didn't say this before, but i think having dob required is probably good